linode/manager
# upcoming: [M3-7869] - Linode Create Refactor - Part 2 - Images and Distributions

## Description 📝

Part 2 of the Linode Create refactor effort 🎉 

## Changes  🔄
- Introduces **Tabs** (Distribution, Marketplace , StackScripts, Backups, Clone)
- Introduces some **helpers for managing query params** (Linode Create relies somewhat heavily on query params)
- Completely **implements the Distribution and Image Tabs**
- Implements a new **`ImageSelectv2`**
  - Yes, this does expand this scope of this project, and yes, it does add a 3rd ImageSelect to the codebase but we'd eventually write this code anyway. The current LinodeSelect literally has a `Paper` baked into it 🙄 
- Adds `LandingHeader` to match the existing Linode Create
- Switches use of `watch` to `useWatch` to fix other components re-rendering when changing `region`
- Configure MUI Autocomplete Group Label theme so that it looks more like our old component

## Preview 📷

Putting a before and after so you can how the refactor is progressing 🤩

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-03-13 at 8 07 33 PM](https://github.com/linode/manager/assets/115251059/e20ab9a8-d4f5-49cc-a672-6dd159a64fd7) | ![Screenshot 2024-03-13 at 8 07 04 PM](https://github.com/linode/manager/assets/115251059/c49ee639-7a83-4970-828e-484414d1e663) |

## How to test 🧪

- Manually test the new functionality
- Check for unnecessary renders
- Look and my code and roast it 😳
  - I want this new code to be super clean and maintainable. If I'm doing something questionable, please call me out! 📣 

## As an Author I have considered 🤔
